torchbenchmark/models/BERT_pytorch/__init__.py

```diff
@@ -58,6 +58,16 @@ def __init__(self, device=None, jit=False):
     def get_module(self):
         return self.trainer.model, self.example_inputs
 
+    def set_eval(self):
+        self.set_mode(False)
+
+    def set_train(self):
+        self.set_mode(True)
+
+    def set_mode(self, train):
+        (model, _) = self.get_module()
+        model.train(train)
```

Comment:
wait, i'm confused why you added these API overrides for (every?) model?  i thought you would only implement this logic once in the base class, and also special logic in any particular model that doesn't work.  You seem to violate your own suggestion that it is not needed to override in ADDING_MODELS.md?

Reply:
oh...i just realized that we don't actually have a base class for the 'Model' each model implements.  i'm not sure if there is a good reason for this.. i thought we did at one point?

Reply:
I initially intended to add a base class and add this logic to it. Then when you suggested to `Model`s in `__init__.py` I assumed you didn't want the base class. I'd rather add a base class. Let me update the PR

Reply:
Oh, I'm really sorry! Sounds like we both confused each other.  I think it's clear now :)